windows/ReactTestApp/Manifest.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <nlohmann/json.hpp>
```

Comment:
We can move `#include <nlohmann/json.hpp>` to the `.cpp` file since it is unused here. Then add the headers we do need:
```suggestion
#include <map>
#include <optional>
#include <string>
#include <vector>
```